doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -66,16 +64,14 @@ The procedure to install CORTX on OVA is mentioned below.
    #. Record the MAC addresses and go to the following directory:
 
       * **cd /etc/sysconfig/network_scripts/**
-      * **vi ifcfg-ens192**
+      * **vi ifcfg-ens32**
       * Add a new line under **BOOTPROTO=dhcp**
-      * Add a new parameter with the MAC Address *HWADDR=<enp0s3-MAC-Address>*
+      * Add a new parameter with the MAC Address *HWADDR=<MAC-Address>*
       * Repeat the steps for enp0s8 and enp0s9 respectively
-      * **vi ifcfg-ens224**
```

Comment:
Do we need to add these commands for new interface names.

Reply:
these interfaces are by default populated in VirtualBox then thereafter we have configure or tweet to reflect the actual interfaces